main.cpp

```diff
@@ -228,346 +305,521 @@ struct University
         std::string hobby = "Play Bass";
         int semestralCredits = 9;
 
-        float weekstudyTime(int research = 28, int teaching = 16);
-        float computeMonthlyExpenses(float food = 250, 
+        float weekStudyTime(int research = 28, int teaching = 16);
```

Comment:
Add a verb to this function name.  `computeWeekStudyTime` is a good choice
- [ ] revise